src/Stratis.Bitcoin/Controllers/ProofOfStakeAttribute.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Stratis.Bitcoin.Controllers
+{
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class)]
+    public class ProofOfStakeAttribute : Attribute
```

Comment:
Can you please add some documentation here to show where this and ProofOfWorkAttribute are intended to be used?